src/nodes/math/MathNode.js

```diff
@@ -152,6 +152,10 @@ class MathNode extends TempNode {
 
 			return builder.changeComponentType( this.aNode.getNodeType( builder ), 'bool' );
 
+		} else if ( method === MathNode.BITCAST ) {
+
+			return this.bNode;
```

Comment:
This should return a string type.

Reply:
We have other nodes that have string inputs but do not have the `*Node` suffix. There is no intention to have values ​​other than nodes for these `*Node` suffixes except for Materials that were automatically converted to `nodeObject`; this should not be a good practice, as it often creates difficult-to-detect errors in complex systems. Create `BitcastNode` for the cleanest solution yet.